codex-rs/core/src/codex.rs

```diff
@@ -3271,11 +3277,28 @@ mod tests {
     use mcp_types::ContentBlock;
     use mcp_types::TextContent;
     use pretty_assertions::assert_eq;
+    use serde::Deserialize;
     use serde_json::json;
     use std::path::PathBuf;
     use std::sync::Arc;
     use std::time::Duration as StdDuration;
 
+    fn test_echo_command() -> Vec<String> {
```

Comment:
This is a helper function that is used way later. It only appears to be used in `rejects_escalated_permissions_when_policy_not_on_request()`, so why not inline it there?